api/internal/handler/handler_test.go

```diff
@@ -38,7 +38,7 @@ func TestSpecCodeResponse(t *testing.T) {
 	resp = SpecCodeResponse(err)
 	assert.Equal(t, &data.SpecCodeResponse{StatusCode: http.StatusNotFound}, resp)
 
-	err = errors.New("system error")
+	err = errors.New("system_config error")
```

Comment:
why change this?

Reply:
Oh, it was a mistake, sorry!

> why change this?

Oh, it was a mistake, sorry!

Reply:
Oh, it was a mistake, sorry!